File: src/main/java/kitchenpos/eatinorders/tobe/domain/JpaEatInOrderRepository.java
package kitchenpos.eatinorders.tobe.domain;

import kitchenpos.eatinorders.domain.OrderStatus;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface JpaEatInOrderRepository extends EatInOrderRepository, JpaRepository<EatInOrder, UUID> {
}

File: src/main/java/kitchenpos/eatinorders/tobe/domain/JpaEatInOrderRepository.kt
package kitchenpos.eatinorders.tobe.domain

import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface JpaEatInOrderRepository : JpaRepository<EatInOrder, UUID>, EatInOrderRepository {
    override fun existsByOrderTableIdAndStatusNot(orderTableId: UUID, status: EatInOrderStatus): Boolean
}
